### PR TITLE
Create basic web application that can read/write a table.

### PR DESCRIPTION
**Pull request by fenghaolw**
Running `node server/http.js` will start a HTTP server listening to the following path:

- `GET /`: serves `index.html` which is a basic HTML page that contains a form to submit new rater and a button to show all raters.
- `GET /content.js`: serves the necessary javascript for the client. For now, it only shows the raters in the page.
- `GET /showRater`: returns the serialized list of raters in the table.
- `POST /addRater`: collects the data in the input fields and insert a corresponding entry to the rater table.

Also use a localhost instance for testing. I feel like it is better than exhibit the actual server in the codebase.

### CODE REVIEW

File: data_store/db.js [hunk 1]
```diff
@@ -1,33 +1,38 @@
-const events = require('events');
 const mysql = require('mysql');
 
-const eventEmitter = new events.EventEmitter();
-
 const con = mysql.createConnection({
-  host: 'rater-dev.cwvjfre1hpn6.us-west-1.rds.amazonaws.com',
+  host: 'localhost',
```
Comment (xyrhythm) [inline data_store/db.js]: Could you keep the aws one in comment? Were you able to connect after the acl change?

File: data_store/db.js [hunk 2]
```diff
@@ -1,33 +1,38 @@
...
     }
-    console.log(result);
+    if (callback) {
+      callback(result);
+    }
+  });
+}
+
+function viewTable(tableName, callback) {
+  querySql(`SELECT * FROM ${tableName}`, '', function(result) {
```
Comment (xyrhythm) [inline data_store/db.js]: add "limit 100" here?

File: server/content.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+function showRater() {
+  const xhr = new XMLHttpRequest();
+  // TODO(fenghaolw): Use url param to pass table name? security concern?
```
Comment (xyrhythm) [inline server/content.js]: Is it possible to make sure that some urls are guarded and only accessible for some logged-in accounts? Agree that it is a TODO.

File: data_store/db.js (hunk 1 above)
Comment (fenghaolw) [inline data_store/db.js]: Reverted this.

File: server/content.js (hunk 3 above)
Comment (fenghaolw) [inline server/content.js]: Yeah that will be a larger TODO in the server/http.js. This method is a handler for clicking the "show rater" button so there is nothing we can control here.

File: data_store/db.js (hunk 2 above)
Comment (fenghaolw) [inline data_store/db.js]: Done.

**Pull request merged**